spark/spark-3.3/pom.xml

```diff
@@ -98,6 +97,36 @@
             <groupId>org.locationtech.jts</groupId>
             <artifactId>jts-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
```

Comment:
Is there a reason why these dependencies only appear in the `spark-3.3` profile?

Reply:
test testing loading from s3 (I used minio with test containers) is only in spark 3.3. I can duplicate it to other versions of spark as well

Reply:
Please move these to the `spark-common` pom.xml so all Spark 3.X pom.xml will share it

Reply:
I think deps with test scope is not propagating when you put it in common and I would avoid putting it as normal deps, I copied them for each of the spark version.